blueocean-pipeline-api-impl/src/test/java/io/jenkins/blueocean/rest/impl/pipeline/PipelineBaseTest.java

```diff
@@ -54,12 +56,18 @@
 public abstract class PipelineBaseTest{
     private static  final Logger LOGGER = LoggerFactory.getLogger(PipelineBaseTest.class);
 
-    public PipelineBaseTest() {
+    @BeforeClass
+    public static void enableJWT() {
```

Comment:
@michaelneale FYI this could have been the cause of some of the 403 vs 200 failures (although I have not verified this at all)